pkg/cmd/roachtest/tests/restore.go

```diff
@@ -290,141 +286,100 @@ func registerRestore(r registry.Registry) {
-			timeout:                24 * time.Hour,
-			suites:                 registry.Suites(registry.Weekly),
-			restoreUptoIncremental: defaultRestoreUptoIncremental,
+			// The weekly 20TB Restore test.
+			hardware:       makeHardwareSpecs(hardwareSpecs{nodes: 15, cpus: 16, volumeSize: 5000}),
+			backup:         backupSpecs{cloud: spec.GCE, fixture: LargeFixture},
+			fullBackupOnly: false,
+			timeout:        24 * time.Hour,
+			suites:         registry.Suites(registry.Weekly),
 		},
+		// Following three tests are just used to benchmark classic restore against
+		// OR with the exact same fixtures and hardware.
 		{
-			// The weekly 32TB, 400 incremental layer Restore test on AWS.
-			//
-			// NB: Prior to 23.1, restore would OOM on backups that had many
-			// incremental layers and many import spans. Together with having a 400
-			// incremental chain, this regression tests against the OOMs that we've
-			// seen in previous versions.
-			//
-			// NB 2: This test sets backup.restore_span.max_file_count to allow for
-			// restore span entries to extend. As of #119840, restore limits the
-			// number of files per restore span entry. When this file cap is less than
-			// the number of inc layers, restore slows signficantly, as the restore
-			// span entries become much smaller.
-			hardware: makeHardwareSpecs(hardwareSpecs{nodes: 15, cpus: 16, volumeSize: 5000,
-				ebsThroughput: 250 /* MB/s */}),
-			backup: makeRestoringBackupSpecs(backupSpecs{
-				version:           "v22.2.4",
-				workload:          tpceRestore{customers: 2000000},
-				numBackupsInChain: 400,
-			}),
-			setUpStmts:             []string{"SET CLUSTER SETTING backup.restore_span.max_file_count = 800"},
-			timeout:                30 * time.Hour,
-			suites:                 registry.Suites(registry.Weekly),
-			restoreUptoIncremental: 400,
+			hardware:       makeHardwareSpecs(hardwareSpecs{}),
+			backup:         backupSpecs{cloud: spec.GCE, fixture: SmallFixture},
+			timeout:        1 * time.Hour,
+			suites:         registry.Suites(registry.Nightly),
+			fullBackupOnly: true,
+			skip:           "used for adhoc benchmarking against OR",
 		},
 		{
-			// The weekly 32TB, 400 incremental layer Restore test on GCP.
-			hardware: makeHardwareSpecs(hardwareSpecs{nodes: 15, cpus: 16, volumeSize: 5000}),
-			backup: makeRestoringBackupSpecs(backupSpecs{
-				version:           "v22.2.4",
-				workload:          tpceRestore{customers: 2000000},
-				cloud:             spec.GCE,
-				numBackupsInChain: 400,
-			}),
-			timeout:                30 * time.Hour,
-			suites:                 registry.Suites(registry.Weekly),
-			restoreUptoIncremental: 400,
-			skip:                   "a recent gcp pricing policy makes this test very expensive. unskip after #111371 is addressed",
+			hardware:       makeHardwareSpecs(hardwareSpecs{}),
+			backup:         backupSpecs{cloud: spec.GCE, fixture: SmallFixture},
+			fullBackupOnly: false,
```

Comment:
oh, i meant to explicitly specify it just for this test which looks identical to the one above